play-v27/src/main/scala/com/gu/googleauth/actions.scala

```diff
@@ -224,7 +224,7 @@ trait Filters extends UserIdentifier with Logging {
     */
   def requireGroup[R[_] <: RequestHeader](
     includedGroups: Set[String],
-    notInValidGroup: R[_] => Result = (_: R[_])  => Forbidden
+    notInValidGroup: RequestHeader => Result = _  => Forbidden
```

Comment:
The Scala 3 compiler no longer tolerates `notInValidGroup: R[_] => Result` - we get this error if we leave it like that:

```
[error] -- [E043] Type Error: /Users/Roberto_Tyley/code/play-googleauth/play-v29/src/main/scala/com/gu/googleauth/actions.scala:227:21
[error] 227 |    notInValidGroup: R[_] => Result = (_: R[_])  => Forbidden
[error]     |                     ^^^^
[error]     | unreducible application of higher-kinded type R to wildcard arguments
[error]     |
[error]     | longer explanation available when compiling with `-explain`
```

The ([details on error `E043`](https://docs.scala-lang.org/scala3/guides/migration/incompat-other-changes.html#wildcard-type-argument)) suggest a couple of workarounds, including a wrapper class, but I don't think that'll work - we should be ok to use the simpler type-parameter-free `RequestHeader`, I can't see any repos at the Guardian that are using the detailed type of the request.